src/csl/mod.rs

```diff
@@ -1555,10 +1566,17 @@ impl<'a> StyleContext<'a> {
                 locale.fallback()
             };
 
+            // First, we lookup for item language
+            if let Some(output) = lookup(resource, item_locale) {
+                return Some(output);
+            }
```

Comment:
This will fail if the citation style contains unconditional local overrides. This if would apply them to all entries without a language. Instead, it should be checked that `item_locale` is `Some` and then do this lookup.

Reply:
Ok, got you.